docs/processes/release/tasks/standards.md

```diff
@@ -0,0 +1,72 @@
+<!--
+© 2021 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# Standards Updates
+
+## Time Zones
+
+Make sure the timezone (Olson) data is up to date.
+
+Olson timezone data should be regularly updated in the trunk. If the data is out
+of date, follow the instruction described in the
+[readme (BROKEN LINK)](http://source.icu-project.org/repos/icu/data/trunk/tzdata/readme.txt)
```

Comment:
svn links to .../repos/icu/data/... are in a separate repo: https://github.com/unicode-org/icu-data

This one is https://github.com/unicode-org/icu-data/blob/master/tzdata/readme.txt

Reply:
Done.